### ISSUE TITLE
Long duplicate text in Understanding 1.4.3

### ISSUE THREAD

**Issue by plannero** [w3c/wcag#1406]
The first 3 long paragraphs of text under "Rationale for the Ratios Chosen" are identical to three paragraphs under "Intent", except for the last few sentences of the note. Probably should be removed from "Intent" in order not to confuse readers.

I am referring to this instance of the document:
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

Minor comment at the same time: You might want to update the reference link to "The American Printing House for the Blind Guidelines for Large Printing" since the url is now redirecting (and redirections sometimes disappear). New url: https://www.aph.org/aph-guidelines-for-print-document-design/ The written content seems to be intact even though url and design has changed.

**Comment by patrickhlauke** [comment]
The first part (duplicate text) seems to have been resolved. The "new url" mentioned in the second part seems to 404, while the current/old link works fine.
